types.go

```diff
@@ -0,0 +1,25 @@
+package helmclient
+
+// Chart returns information about a Helm Chart.
+type Chart struct {
+	// Version is the version of the Helm Chart.
+	Version string
+}
+
```

Comment:
Are the structs below used?

Reply:
Yes they are used by GetReleaseContent and GetReleaseHistory.

Reply:
Ah sure they are moved from `spec.go`. I haven't noticed that in the first place 👍